.prettierrc.js

```diff
@@ -1,3 +1,33 @@
+const tailwindPlugin = require('tailwind-plugin');
+const importsPlugin = require('imports-plugin');
+
+/**
+ * Defining the plugins this way is a workaround as the tailwind & the imports plugin wouldn't work otherwise when
+ * installed together.
+ *
+ * General info about the issue:
+ *   - https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/31
+ *   - https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/9
+ *
+ * Details about the workaround:
+ *   - Base: https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/9#issuecomment-1021028722
+ *   - Addition: https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/9#issuecomment-1157359437
+ */
+const plugins = [
+  {
+    parsers: {
+      typescript: {
+        ...tailwindPlugin.parsers.typescript,
+        preprocess: importsPlugin.parsers.typescript.preprocess,
+      },
+    },
+    options: {
+      ...tailwindPlugin.options,
+      ...importsPlugin.options,
+    },
+  },
+];
```

Comment:
FYI, the [comment providing the workaround](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/9#issuecomment-1021028722) suggests putting this into a dedicated file and importing it here, but I did not find a way to make this work with our prettier config package as consuming code wasn't able to find the imported file then, no matter what kind of "path resolution mechanism" i tried (`./file.js`, `/file.js`, `file.js`, ...) 🤷‍♂️.

As this file is still rather small, inlining this is IMHO not a problem and I didn't want to spend too much time on this.

Reply:
I would also say that it doesn't matter and one sees right away, that there's a workaround going on.

My first thought regarding your issue: Did you check if the other file was actually added to the package?
Or in other words, did you try to add it to `files` in `package.json`?

Reply:
Yep, I added it to `files` & it was in the `node_modules`, but path resolution didn't work 🤷‍♂️